src/components/Sidenav/SidenavBase.style.ts

```diff
@@ -79,7 +79,8 @@ export const ButtonGroup = styled.div`
   display: flex;
   flex-direction: column;
   justify-content: flex-end;
-  a:not(:first-of-type) {
+  a:not(:first-of-type),
+  button:first-of-type {
```

Comment:
```suggestion
  * + * {
```

I think this will solve it?